contactsummary/js/src/contact-summary/services/contactDetails.js

```diff
@@ -86,6 +86,28 @@ define([
             return deferred.promise;
         }
 
+      /**
```

Comment:
The indentation is wrong, and you need to specify the type of the param, and the types are always written as {types} in js docs
```js
/**
 * Calculate age from birth date
 *
 * @param {object} dateOfBirth Date of birth in a YYYY-MM-DD format
 * @return {string}
 */
function